CONTRIBUTING.md

```diff
@@ -81,13 +85,22 @@ so do familiarize yourself with the following guidelines.
 
 ## Environment requirements
 
-JDK >= 11 referred to by the `JAVA_HOME` environment variable.
+* JDK >= 11 referred to by the `JAVA_HOME` environment variable.
+
+  * Note, any version above 11 should work in theory, but JDK 11 is the only version we test with,
```

Comment:
what's with the formatting here?

Reply:
It's a double indent 
* something
  * like this